src/core/highlight-vars.js

```diff
@@ -0,0 +1,29 @@
+// Module core/highlight-vars
+// Highlights occurrences of a <var> within a .algorithm on click
+// if a <var> is set only once (unused/undeclared), it's highlighted with class ".bug"
+
+import hlVars from "deps/text!core/css/var.css";
+const styleElement = document.createElement("style");
+styleElement.textContent = hlVars;
+styleElement.classList.add("removeOnSave");
+document.head.appendChild(styleElement);
+
+export const name = "core/highlight-vars";
+
+export function run(conf, doc, cb) {
+  for (const elem of document.querySelectorAll(".algorithm var")) {
```

Comment:
so, taking a look at, for instance:
https://w3c.github.io/payment-request/#constructor

The some of the `<var>`s come from the `<p>` that defines the algorithm. For example:

```HTML
<p>
  The <code><dfn>PaymentRequest(<var>methodData</var>,
  <var>details</var>, <var>options</var>)</dfn></code> constructor <em class="rfc2119" title="MUST">MUST</em> act as follows:
</p>
```

So maybe we should match on each `section`?

Reply:
Yes sir. I think we should highlight matches in the entire `<section>`. I took the "algorithm"  stated in issue too literally. Haha.

``` js
function highlightVars({ target }) {
  const value = target.innerText;
  const parent = target.closest(".algorithm"); // <<< need a match here also. to closest section
```

Will fix.
Currently working on different highlight colors.